numba/testing.py

```diff
@@ -1,9 +1,522 @@
 from __future__ import print_function, division, absolute_import
 
 import sys
+import os
 import functools
-
+import numba.unittest_support as unittest
```

Comment:
As a style issue, the PEP 8 recommendation for imports is:
> Imports should be grouped in the following order:

> -    standard library imports
-  related third party imports
-  local application/library specific imports

> You should put a blank line between each group of imports. 

I like this rule as it makes dependencies much easier to figure out.